app/docs/index-files/index-20.html

```diff
@@ -676,7 +676,7 @@ <h2 class="title">T</h2>
 <dd>&nbsp;</dd>
 <dt><span class="memberNameLink"><a href="../nu/yona/app/R.dimen.html#thirty_two">thirty_two</a></span> - Static variable in class nu.yona.app.<a href="../nu/yona/app/R.dimen.html" title="class in nu.yona.app">R.dimen</a></dt>
 <dd>&nbsp;</dd>
-<dt><span class="memberNameLink"><a href="../nu/yona/app/utils/AppUtils.html#throwException-java.lang.String-java.lang.Exception-java.lang.Thread-nu.yona.app.listener.DataLoadListener-">throwException(String, Exception, Thread, DataLoadListener)</a></span> - Static method in class nu.yona.app.utils.<a href="../nu/yona/app/utils/AppUtils.html" title="class in nu.yona.app.utils">AppUtils</a></dt>
+<dt><span class="memberNameLink"><a href="../nu/yona/app/utils/AppUtils.html#reportException-java.lang.String-java.lang.Exception-java.lang.Thread-nu.yona.app.listener.DataLoadListener-">reportException(String, Exception, Thread, DataLoadListener)</a></span> - Static method in class nu.yona.app.utils.<a href="../nu/yona/app/utils/AppUtils.html" title="class in nu.yona.app.utils">AppUtils</a></dt>
```

Comment:
What file is it? Is it build-time-generated? If so, it shouldn't be in Git.

Reply:
These html files are documentation files generated by Android Studio based on the comments added in the .java files for classes and methods. 
These files are not build-time-generated.